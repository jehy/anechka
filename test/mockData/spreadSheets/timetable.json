{
  "config": {
    "method": "GET",
    "headers": {
      "Accept-Encoding": "gzip",
      "User-Agent": "google-api-nodejs-client/0.7.2 (gzip)",
      "Accept": "application/json"
    },
    "params": {},
    "responseType": "json"
  },
  "data": {
    "range": "timetable_2019!A1:Z33",
    "majorDimension": "ROWS",
    "values": [
      [
        "1",
        "январь",
        "2",
        "февраль",
        "3",
        "март",
        "4",
        "апрель",
        "5",
        "май"
      ],
      [],
      [
        "1",
        "Ivanov",
        "1",
        "Sidorov",
        "1",
        "Smith",
        "1",
        "Smith",
        "1",
        "Ivanov"
      ],
      [
        "2",
        "Ivanov",
        "2",
        "",
        "2",
        "",
        "2",
        "Redfield",
        "2",
        "Ivanov"
      ],
      [
        "3",
        "Ivanov",
        "3",
        "",
        "3",
        "",
        "3",
        "Petrov",
        "3",
        "Ivanov"
      ],
      [
        "4",
        "Ivanov",
        "4",
        "Smith",
        "4",
        "Petrov",
        "4",
        "Sidorov",
        "4",
        "Ivanov"
      ],
      [
        "5",
        "Ivanov",
        "5",
        "Redfield",
        "5",
        "Redfield",
        "5",
        "Ivanov",
        "5",
        "Ivanov"
      ],
      [
        "6",
        "Ivanov",
        "6",
        "Petrov",
        "6",
        "Petrov",
        "6",
        "",
        "6",
        "Redfield"
      ],
      [
        "7",
        "Ivanov",
        "7",
        "Ivanov",
        "7",
        "Ivanov",
        "7",
        "",
        "7",
        "Petrov"
      ],
      [
        "8",
        "Ivanov",
        "8",
        "Sidorov",
        "8",
        "Sidorov",
        "8",
        "Smith",
        "8",
        "Smith"
      ],
      [
        "9",
        "Petrov",
        "9",
        "",
        "9",
        "",
        "9",
        "Redfield",
        "9",
        "Ivanov"
      ],
      [
        "10",
        "Sidorov",
        "10",
        "",
        "10",
        "",
        "10",
        "Petrov",
        "10",
        "Ivanov"
      ],
      [
        "11",
        "Ivanov",
        "11",
        "Smith",
        "11",
        "Smith",
        "11",
        "Ivanov",
        "11",
        "Ivanov"
      ],
      [
        "12",
        "",
        "12",
        "Redfield",
        "12",
        "Redfield",
        "12",
        "Sidorov",
        "12",
        "Ivanov"
      ],
      [
        "13",
        "",
        "13",
        "Petrov",
        "13",
        "Petrov",
        "13",
        "",
        "13",
        "Redfield"
      ],
      [
        "14",
        "Smith",
        "14",
        "Ivanov",
        "14",
        "Ivanov",
        "14",
        "",
        "14",
        "Petrov"
      ],
      [
        "15",
        "Redfield",
        "15",
        "Smith",
        "15",
        "Sidorov",
        "15",
        "Smith",
        "15",
        "Smith"
      ],
      [
        "16",
        "Petrov",
        "16",
        "",
        "16",
        "",
        "16",
        "Redfield",
        "16",
        "Ivanov"
      ],
      [
        "17",
        "Sidorov",
        "17",
        "",
        "17",
        "",
        "17",
        "Petrov",
        "17",
        "Redfield"
      ],
      [
        "18",
        "Ivanov",
        "18",
        "Sidorov",
        "18",
        "Smith",
        "18",
        "Ivanov",
        "18",
        "Ivanov"
      ],
      [
        "19",
        "",
        "19",
        "Redfield",
        "19",
        "Redfield",
        "19",
        "Smith",
        "19",
        "Ivanov"
      ],
      [
        "20",
        "",
        "20",
        "Ivanov",
        "20",
        "Petrov",
        "20",
        "",
        "20",
        "Petrov"
      ],
      [
        "21",
        "Smith",
        "21",
        "Sidorov",
        "21",
        "Ivanov",
        "21",
        "",
        "21",
        "Smith"
      ],
      [
        "22",
        "Redfield",
        "22",
        "Smith",
        "22",
        "Sidorov",
        "22",
        "Redfield",
        "22",
        "Ivanov"
      ],
      [
        "23",
        "Smith",
        "23",
        "",
        "23",
        "",
        "23",
        "Petrov",
        "23",
        "Redfield"
      ],
      [
        "24",
        "Sidorov",
        "24",
        "",
        "24",
        "",
        "24",
        "Ivanov",
        "24",
        "Petrov"
      ],
      [
        "25",
        "Ivanov",
        "25",
        "Petrov",
        "25",
        "Smith",
        "25",
        "Smith",
        "25",
        "Ivanov"
      ],
      [
        "26",
        "",
        "26",
        "Redfield",
        "26",
        "Redfield",
        "26",
        "Redfield",
        "26",
        "Ivanov"
      ],
      [
        "27",
        "",
        "27",
        "Ivanov",
        "27",
        "Petrov",
        "27",
        "",
        "27",
        "Smith"
      ],
      [
        "28",
        "Smith",
        "28",
        "Sidorov",
        "28",
        "Sidorov",
        "28",
        "",
        "28",
        "Ivanov"
      ],
      [
        "29",
        "Redfield",
        "29",
        "",
        "29",
        "Ivanov",
        "29",
        "Petrov",
        "29",
        "Redfield"
      ],
      [
        "30",
        "Petrov",
        "30",
        "",
        "30",
        "",
        "30",
        "Ivanov",
        "30",
        "Petrov"
      ],
      [
        "31",
        "Sidorov",
        "31",
        "",
        "31",
        "",
        "",
        "",
        "31",
        "Smith"
      ]
    ]
  },
  "headers": {
    "alt-svc": "quic=\":443\"; ma=2592000; v=\"46,44,43,39\"",
    "cache-control": "private",
    "connection": "close",
    "content-encoding": "gzip",
    "content-type": "application/json; charset=UTF-8",
    "date": "Tue, 14 May 2019 11:16:32 GMT",
    "server": "ESF",
    "transfer-encoding": "chunked",
    "vary": "Origin, X-Origin, Referer",
    "x-frame-options": "SAMEORIGIN",
    "x-xss-protection": "0"
  },
  "status": 200,
  "statusText": "OK"
}
