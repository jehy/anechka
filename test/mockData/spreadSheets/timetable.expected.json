{
  "id": {
    "2019": {
      "1": {
        "1": "Ivanov",
        "2": "Ivanov",
        "3": "Ivanov",
        "4": "Ivanov",
        "5": "Ivanov",
        "6": "Ivanov",
        "7": "Ivanov",
        "8": "Ivanov",
        "9": "Petrov",
        "10": "Sidorov",
        "11": "Ivanov",
        "14": "Smith",
        "15": "Redfield",
        "16": "Petrov",
        "17": "Sidorov",
        "18": "Ivanov",
        "21": "Smith",
        "22": "Redfield",
        "23": "Smith",
        "24": "Sidorov",
        "25": "Ivanov",
        "28": "Smith",
        "29": "Redfield",
        "30": "Petrov",
        "31": "Sidorov"
      },
      "2": {
        "1": "Sidorov",
        "4": "Smith",
        "5": "Redfield",
        "6": "Petrov",
        "7": "Ivanov",
        "8": "Sidorov",
        "11": "Smith",
        "12": "Redfield",
        "13": "Petrov",
        "14": "Ivanov",
        "15": "Smith",
        "18": "Sidorov",
        "19": "Redfield",
        "20": "Ivanov",
        "21": "Sidorov",
        "22": "Smith",
        "25": "Petrov",
        "26": "Redfield",
        "27": "Ivanov",
        "28": "Sidorov"
      },
      "3": {
        "1": "Smith",
        "4": "Petrov",
        "5": "Redfield",
        "6": "Petrov",
        "7": "Ivanov",
        "8": "Sidorov",
        "11": "Smith",
        "12": "Redfield",
        "13": "Petrov",
        "14": "Ivanov",
        "15": "Sidorov",
        "18": "Smith",
        "19": "Redfield",
        "20": "Petrov",
        "21": "Ivanov",
        "22": "Sidorov",
        "25": "Smith",
        "26": "Redfield",
        "27": "Petrov",
        "28": "Sidorov",
        "29": "Ivanov"
      },
      "4": {
        "1": "Smith",
        "2": "Redfield",
        "3": "Petrov",
        "4": "Sidorov",
        "5": "Ivanov",
        "8": "Smith",
        "9": "Redfield",
        "10": "Petrov",
        "11": "Ivanov",
        "12": "Sidorov",
        "15": "Smith",
        "16": "Redfield",
        "17": "Petrov",
        "18": "Ivanov",
        "19": "Smith",
        "22": "Redfield",
        "23": "Petrov",
        "24": "Ivanov",
        "25": "Smith",
        "26": "Redfield",
        "29": "Petrov",
        "30": "Ivanov"
      },
      "5": {
        "1": "Ivanov",
        "2": "Ivanov",
        "3": "Ivanov",
        "4": "Ivanov",
        "5": "Ivanov",
        "6": "Redfield",
        "7": "Petrov",
        "8": "Smith",
        "9": "Ivanov",
        "10": "Ivanov",
        "11": "Ivanov",
        "12": "Ivanov",
        "13": "Redfield",
        "14": "Petrov",
        "15": "Smith",
        "16": "Ivanov",
        "17": "Redfield",
        "18": "Ivanov",
        "19": "Ivanov",
        "20": "Petrov",
        "21": "Smith",
        "22": "Ivanov",
        "23": "Redfield",
        "24": "Petrov",
        "25": "Ivanov",
        "26": "Ivanov",
        "27": "Smith",
        "28": "Ivanov",
        "29": "Redfield",
        "30": "Petrov",
        "31": "Smith"
      }
    }
  },
  "lastUpdate": "2019-05-14T11:39:32.124Z"
}
